src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -36,6 +40,8 @@ public enum IamAction {
   // snapshots
   UPDATE_SNAPSHOT,
   READ_DATA,
+  READ_AGGREGATE_DATA,
+  CREATE_SNAPSHOT_REQUEST,
   DISCOVER_DATA,
```

Comment:
I don't see the settings actions
```
      "get_snapshot_builder_settings" = {
        description = "Can get the snapshot builder settings for this snapshot"
      }
      "update_snapshot_builder_settings" = {
        description = "Can update the snapshot builder settings for this snapshot"
      }
```

Reply:
The action is already there because it was added for datasets.